### ISSUE TITLE
namespace assignment not consistent

### ISSUE THREAD

**Issue by AramKrol** [obophenotype/cell-ontology#531]
cell concepts missing namespace definition, eg CL:0011114
namespace: cell

**Comment by paolaroncaglia** [comment]
The issue reported above (cell concepts missing namespace definition, eg CL:0011114; namespace: cell) is still present. I don't know how widespread it is. As far as I know, the AP has_obo_namespace is not a mandatory one, hence I presume that there are no checks in place to flag terms missing a namespace. Should we have such a check?

**Comment by anitacaron** [comment]
Two alternatives:
1. SPARQL CONSTRUCT to add missing oboInOwl:hasOBONamespace https://api.triplydb.com/s/VaQ-4yPBP. Downside: Require running an ODK command (reduce accessibility) 
2. Delete the namespace annotations for this property

**Comment by matentzn** [comment]
Rather than create another technical burden on curators and infrastructure, I vote to remove this tag.

**Comment by dosumis** [comment]
Agree, we should remove. AFAIK, the only ontology where obo namespace is useful is GO. If anyone using CL still needs it they should have complained by now given how many terms lack it .

**Comment by matentzn** [comment]
@anitacaron You can do this with a quick regex hack I think, but don't do it this week (give other people a chance to comment)

**Comment by cmungall** [comment]
agreed

the original obo namespace conflated two useful things: (1) what ontology is this term in (2) what kind of thing is this

(1) is best served by the current proposal to ensure every term is tagged by rdfs:isDefinedBy
(2) is best served by having a category annotation on each term (ubergraph does this)
